index.js

```diff
@@ -1,33 +1,32 @@
 
-
-//Getting the different placeholders
-
 let content = document.getElementById("content");
 let integerValue = parseInt(content.innerHTML);
 
-const decreaseButton = document.getElementById("btn-decrease");
-const resetButton = document.getElementById("btn-reset");
-const increaseButton = document.getElementById("btn-increase");
-
-
-//Increasing value when the increase button is clicked on
-increaseButton.addEventListener("click", () => { 
-    integerValue += 1;
-    content.innerHTML = integerValue;
+const arrayButton = document.getElementsByTagName("button");
```

Comment:
You might have other button in the app so a better solution is to add the same class to each button and then use getElementsByClassName for the HTMLCollection

` <button class="button-event" id="btn-decrease">Decrease</button>`

`<button class="button-event" id="btn-reset">Reset</button>`

`<button class="button-event" id="btn-increase">Increase</button>`